engine/utils/Logging.h

```diff
@@ -103,7 +124,9 @@ static constexpr const char REPLACE_STRING[] = "{}";
  *                       the text with
  * @param size - the size of variantItems
  */
-static void VariantFormat(String& text, const VariantContainer* variantItems, const size_t& size)
+[[maybe_unused]] static void VariantFormat(String& text,
+                                           const VariantContainer* variantItems,
+                                           const size_t& size)
```

Comment:
You decided to pay the piper on this one? Fair enough.

Reply:
I might just silence the warning in Clion tbh. This warning was being spammed everywhere in the CLI and it was drowning out all the actual concerns.